specs/interop/predeploys.md

```diff
@@ -131,8 +271,9 @@ A simple implementation of the `validateMessage` function is included below.
 
 ```solidity
     function validateMessage(Identifier calldata _id, bytes32 _msgHash) external {
-        // We need to know if this is being called on a depositTx
-        if (IL1BlockInterop(Predeploys.L1_BLOCK_ATTRIBUTES).isDeposit()) revert NoExecutingDeposits();
+        // TODO calldata load the inputs
+        // TODO compute checksum
+        // TODO check if checksum is known access-list entry
```

Comment:
My understanding is that we can get rid of the `isDeposit()` check now that we've access-lists.

I left the implementation as TODO. I wrote some solidity/yul snippets above in the spec that can largely be used here for implementation, but would like to agree on spec of those before we duplicate any work here in solidity.

Reply:
Lets get rid of the `isDeposit` check as well as the deposit context. We don't need either of them at this point

Reply:
Updated to include a part of the wonderland code, with reference to the relevant spec sections for checksum computation and info about the is-warm check.